app/views/providers/check_provider_answers/index.html.erb

```diff
@@ -61,15 +67,16 @@
         name: :proceeding_type,
         url: providers_legal_aid_application_proceedings_types_path(anchor: 'proceeding-search-input'.freeze),
         question: t('.section_proceeding.proceeding'),
-        answer: @proceeding_types.first.meaning
+        answer: @proceeding_types.first.meaning,
+        read_only: @read_only
       ) %>
 
   </dl>
-  <div class="govuk-!-padding-bottom-9"></div>
 
   <%= next_action_buttons_with_form(
-        url: continue_providers_legal_aid_application_check_provider_answers_path,
-        method: :patch,
-        show_draft: true
+        url: @read_only ? providers_legal_aid_applications_path : continue_providers_legal_aid_application_check_provider_answers_path,
+        method: @read_only ? :get : :patch,
+        show_draft: @read_only ? false : true,
+        continue_button_text: @read_only ? t('generic.back_to_your_applications') : t('generic.continue')
```

Comment:
Again - rather than making every option react to the same tertiary operator, it would be clearer to separate the options into two versions. So:
```erb
<%
  next_button_options = { .... }
  read_only_next_button_options = { ... }
%>
<%= next_action_buttons_with_form(
  @read_only ? read_only_next_button_options : next_button_options
) %>

Reply:
👍